doc/preparing.rst

```diff
@@ -0,0 +1,99 @@
+.. _preparing_repositories:
+
+Preparing a repository for Binder
+=================================
+
+You can do this by making sure your repo contains:
+
+* A collection of Jupyter Notebooks. These notebooks will be made available to
+  users of your Binder.
+* One (or many) text files that specify the requirements of your code. For
+  example, a ``requirements.txt`` or ``environment.yml`` file. See the
+  below examples for a list of supported files and environments.
+
+Verify, and add if needed, the files specified above to the repository.
+Once that's done, navigate to ``mybinder.org`` and insert the URL for
+your git repository. Press ``Launch`` to automatically create your Binder.
+The Binder service will be automatically send you to a live Jupyter session
+connected to this repository.
+
+.. note::
+
+   If a previous version of the repository has already been built, Binder will
+   only build a new one if the git hashes don't match. If Binder When Binder
+   *doesn't* need to build a repository, the process of connecting to the live
+   computational environment is much faster.
+
+
+Simple Python dependencies
+--------------------------
+
+Many repositories already contain a ``requirements.txt`` specifying the
+dependencies of that repository. For 'simple to install' dependencies, a
+``requirements.txt`` should meet your needs. To generate a ``requirements.txt`` from
+the environment you have locally use ``pip freeze > requirements.txt``. This
+will list all packages you have installed, and can be a starting point for
+constructing your ``requirements.txt``. We recommend you only list those
+packages you really need to successfully build the repository.
+
+Take a look at the `binder-examples/requirements <https://github.com/binder-examples/requirements>`_
+repository to see an example.
+
+
+Using conda packages
+--------------------
+
+For 'complex to install' packages, like ``numpy`` or ``scikit-learn``, we
```

Comment:
I think you should pick a different example now :D numpy nor scikit-learn are no longer complex to install with requirements.txt IMO :D MNE perhaps?

Reply:
MNE? If you can think of something that isn't an ipywidget we can change it. I couldn't think of anything off the top of my head. gdal? but then that isn't a python package really.

Ja, damn those wheels ...

Reply:
hands down geopandas. anything in the geospatial analytics world **sucks** to install with pip. Also make sure we use conda-forge.